### DISCUSSION TITLE
Bad quality when resize and crop an image

### DISCUSSION THREAD

**Discussion by zn** [SixLabors/ImageSharp#1627] [Q&A] [answered]
Hello!
I need to resize the image to 325x185. And I also want it to be cropped on center. I first resize and then crop, but the quality of the resulting image is poor. 
Here is my code
```c#
using Image image = await Image.LoadAsync(file);

image.Mutate(i => i.Resize(new ResizeOptions   // resize
{
    Size = size,
    Mode = ResizeMode.Min
})
.Resize(new ResizeOptions                      // crop
{
    Size = size,
    Mode = ResizeMode.Crop,
    Position = AnchorPositionMode.Center
})
```
And here is the image that i'm trying to resize and crop

_Original_
![original](https://user-images.githubusercontent.com/39989294/118082093-3be93300-b3c5-11eb-85a5-2461f9f7246f.jpg)

_325x185_
![0517eed4-9eaf-45a8-8d3f-eb8281b12520](https://user-images.githubusercontent.com/39989294/118082254-84a0ec00-b3c5-11eb-8225-ebebbecbfd52.jpg)

What am I doing wrong?

**Comment by JimBobSquarePants** [comment, marked as answer]
You're resampling the image 2x which is going to cause a loss of information. You don't actually need to do that, `ResizeMode.Crop` gives you the expected output.

With my following test code:
```c#
using (var image = Image.Load<Rgba32>(Path.Combine(inPath, "1627.jpg")))
{
    Size size = new Size(325, 185);
    image.Mutate(i => i.Resize(
        new ResizeOptions
        {
            Size = size,
            Mode = ResizeMode.Crop,
            Position = AnchorPositionMode.Center
        }));

    image.Save(Path.Combine(outPath, "1627.jpg"));
}
```

Before
![118082254-84a0ec00-b3c5-11eb-8225-ebebbecbfd52](https://user-images.githubusercontent.com/385879/118101081-18c38100-b3cf-11eb-883c-ee4aa418fed9.jpg)

After
![1627](https://user-images.githubusercontent.com/385879/118101048-12350980-b3cf-11eb-9548-774e0e763c2f.jpg)

**Reply by zn** [reply]
Thank you!(❁´◡`❁)
